### PR TITLE
[sdk 1.0] fix tests

### PR DESCRIPTION
**Pull request by vladimirvshivkov**
idea is to adopt SDK 1.0 and AOC 2.0
occasionally reworked of css cluster info module and added dns zone info module.

### CODE REVIEW

**Comment by vladimirvshivkov** [comment]
idea is to adopt SDK 1.0 and AOC 2.0

File: plugins/modules/dns_zone_info.py [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+DOCUMENTATION = '''
+module: dns_zone_info
+short_description: Get DNS PTR Records
+extends_documentation_fragment: opentelekomcloud.cloud.otc
+version_added: "0.1.2"
```
Comment (anton-sidelnikov) [inline plugins/modules/dns_zone_info.py:18]: 0.13.1

File: plugins/modules/dns_zone_info.py [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+DOCUMENTATION = '''
+module: dns_zone_info
+short_description: Get DNS PTR Records
```
Comment (anton-sidelnikov) [inline plugins/modules/dns_zone_info.py:16]: wrong description?

File: plugins/modules/dns_zone_info.py [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+extends_documentation_fragment: opentelekomcloud.cloud.otc
+version_added: "0.1.2"
+author: "Vladimir Vshivkov (@vladimirvshivkov)"
+description:
+    - Get DNS PTR Records from the OTC.
```
Comment (anton-sidelnikov) [inline plugins/modules/dns_zone_info.py:21]: here PTR

**Review by anton-sidelnikov — CHANGES_REQUESTED** [review]

**Review by Polina-Gubina — DISMISSED** [review]

File: plugins/modules/css_cluster_info.py [hunk 4]
```diff
@@ -197,33 +198,15 @@ class CSSClusterInfoModule(OTCModule):
...
-            if raw:
-                dt = raw.to_dict()
-                dt.pop('location')
-                data.append(dt)
-        else:
-            for raw in self.conn.css.clusters(**query):
-                dt = raw.to_dict()
-                dt.pop('location')
-                data.append(dt)
+        css_clusters = self.conn.css.find_cluster(**kwargs)
```
Comment (Polina-Gubina) [inline plugins/modules/css_cluster_info.py:205]: what if name is not set and user want to get list by some filter?

**Review by Polina-Gubina — APPROVED** [review]

**Review by anton-sidelnikov — APPROVED** [review]

**Pull request merged**